maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/config/S3Config.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.laa.maat.scheduled.tasks.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+
+@Configuration
+public class S3Config {
+
+    @Value("${aws_region}")
```

Comment:
Is this coming from the `AWS_REGION` environment variable that's set as part of _environment.tpl_ for the pods in the cluster?